integrations/nifi/nifi-nar-bundles/nifi-standard-services/nifi-spark-service-api/src/main/java/com/thinkbiganalytics/nifi/v2/spark/JobService.java

```diff
@@ -0,0 +1,58 @@
+package com.thinkbiganalytics.nifi.v2.spark;
+
+/*-
+ * #%L
+ * thinkbig-nifi-hadoop-service-api
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)